conf/modules.config

```diff
@@ -72,28 +72,33 @@ process {
         ]
     }
 
-    withName: 'BWAMEM2_MEM' {
+    // This BWAMEM2_MEM belongs to the coverage module
+    withName: 'BWAMEM2_MEM_COVERAGE' {
         cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
-        memory = { check_max( 72.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 16.h  * task.attempt, 'time'    ) }
+        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 1.h  * task.attempt,  'time'    ) }
 
         ext.args  = "-M"
-        ext.args2 = [
-            '-f',
-            '12',
-            '-F',
-            '256',
-            '-uS',
-        ].join(' ').trim()
+        ext.args2 = "-F 268 -uS"
     }
 
-    withName: 'SAMTOOLS_BAM2FQ' {
+    /* Decontamination */
+    withName: 'BWAMEM2DECONTNOBAMS' {
         cpus   = { check_max( 2     * task.attempt, 'cpus'    ) }
-        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
         time   = { check_max( 4.h   * task.attempt, 'time'    ) }
-        ext.prefix = "decontaminated"
+        ext.prefix = { "${meta2.id}_decontaminated" }
```

Comment:
I think that one should be ` ext.prefix = "decontaminated"` because you have names like
`human_phix_human_phix_decontaminated`

Reply:
good catch, indeed I'll use "decontaminated". Thanks